.github/workflows/python-validate.yml

```diff
@@ -60,15 +60,15 @@ jobs:
           cd python/packages/sdk
 
           python3 -m pip install black pyproject-fmt
-          python3 -m black --check sdk
+          python3 -m black --check .
           python3 -m pyproject_fmt --indent 4 pyproject.toml
 
       - name: Check code with linter
         run: |
           cd python/packages/sdk
 
           python3 -m pip install ruff
-          python3 -m ruff sdk
+          python3 -m ruff .
```

Comment:
Same here

Reply:
Added a commit that does this with `ruff`